stablehlo/reference/Ops.cpp

```diff
@@ -148,6 +223,17 @@ SmallVector<Tensor> eval(
                                    batchNormInferenceOp.getFeatureIndex(),
                                    batchNormInferenceOp.getType());
       scope.add(op.getResults(), {result});
+    } else if (auto batchNormTrainingOp = dyn_cast<BatchNormTrainingOp>(op)) {
+      auto operand = scope.find(batchNormTrainingOp.getOperand());
+      auto scale = scope.find(batchNormTrainingOp.getScale());
+      auto offset = scope.find(batchNormTrainingOp.getOffset());
+      auto results = evalBatchNormTrainingOp(
+          operand, scale, offset, batchNormTrainingOp.getEpsilon(),
+          batchNormTrainingOp.getFeatureIndex(),
+          {batchNormTrainingOp.getOutput().getType(),
+           batchNormTrainingOp.getBatchMean().getType(),
+           batchNormTrainingOp.getBatchVar().getType()});
+      scope.add(op.getResults(), {results});
```

Comment:
Why do we say `{results}` here if a regular `results` would suffice? Same for other locations in Ops.cpp - apologies for not noticing this in other reviews.